src/js/graphql/Client.ts

```diff
@@ -1,46 +1,64 @@
-import {
-  ApolloClient, HttpLink, InMemoryCache
-} from '@apollo/client'
+import { ApolloClient, from, HttpLink, InMemoryCache } from '@apollo/client'
+import { onError } from '@apollo/client/link/error'
 
 const uri: string = process.env.NEXT_PUBLIC_API_SERVER ?? ''
+const httpLinkPro = new HttpLink({
+  uri: uri
+})
+const errorLink = onError(({ graphQLErrors, networkError }) => {
+  if (graphQLErrors !== null && graphQLErrors !== undefined) {
```

Comment:
to test for nullness and undefined you can just use `if (graphQLErrors != null)`

Reply:
hmm I added the test against `undefined` because `TypeScript` was giving me errors. After deleting it, `yarn build` also gave error
```
Type error: 'graphQLErrors' is possibly 'undefined'.

   8 | const errorLink = onError(({ graphQLErrors, networkError }) => {
   9 |   if (graphQLErrors !== null) {
> 10 |     graphQLErrors.forEach(({ message, locations, path }) => {
     |     ^

```

Reply:
can you try with just 1 equal sign?  != 
Example:  `if (graphQLErrors != null) { ...`

Reply:
1 equal sign passed `yarn build`. Gonna push the changes.